src/repm-invoker.ts

```diff
@@ -70,9 +75,9 @@ export class REPMWasmInvoker {
       // TODO: Have to import dynamically to hide this from Jest.
       // Jest cannot parse the es6 behind this import, I don't know why.
       // TODO: We need to have some way to switch between debug and release.
-      const {default: init, dispatch} = await import(
-        './wasm/debug/replicache_client.js'
-      );
+      // const {default: init, dispatch} = await import(
```

Comment:
Did you mean to check this in? Seems like _dispatch is undefined now.

Reply:
This can be simplified but I want to ensure sample/redo works